proxy/api.go

```diff
@@ -0,0 +1,68 @@
+package proxy
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/ElrondNetwork/notifier-go/config"
+	"github.com/ElrondNetwork/notifier-go/dispatcher"
+	"github.com/ElrondNetwork/notifier-go/proxy/handlers"
+
+	"github.com/gin-gonic/gin"
+)
+
+type webServer struct {
+	router        *gin.Engine
+	notifierHub   dispatcher.Hub
+	generalConfig *config.GeneralConfig
+}
+
+// NewWebServer creates and configures an instance of webServer
+func NewWebServer(generalConfig *config.GeneralConfig) (*webServer, error) {
+	router := gin.Default()
```

Comment:
I think you should also setup cors:
`router.Use(cors.Default())`

Reply:
done